tests/scripts/test-e2e-standalone.sh

```diff
@@ -33,20 +33,20 @@ TMPDIR=$(mktemp -d)
 # ARTIFACT_DIR is defined when running in a prow job
 ARTIFACT_DIR=${ARTIFACT_DIR:-$TMPDIR}
 
-# temp directory for index store
-RAG_TMP_DIR=$(mktemp -d)
-# Download index store
-RAG_INDEX="https://github.com/ilan-pinto/lightspeed-rag-documents/releases/latest/download/local.zip"
-export RAG_INDEX_DIR="$RAG_TMP_DIR/vector-db/ocp-product-docs"
-# Configure index store
-mkdir -p $RAG_INDEX_DIR
-wget $RAG_INDEX
-unzip -j local.zip -d $RAG_INDEX_DIR
-rm -f local.zip
 # configure feedback storage location
 export FEEDBACK_STORAGE_LOCATION="$ARTIFACT_DIR/insights/feedback"
 echo "Feedback storage location: $FEEDBACK_STORAGE_LOCATION"
 
+# temp directory for index store
+RAG_TMP_DIR=$(mktemp -d)
+# Download index store
+export RAG_INDEX_DIR="$RAG_TMP_DIR/vector_db/ocp_product_docs/4.15"
+export RAG_EMBEDDINGS_MODEL_DIR="$RAG_TMP_DIR/embeddings_model"
+podman create --replace --name tmp-rag-container quay.io/openshift/lightspeed-rag-content@sha256:eb3472dd675cc79b02e4d15b53b12c9c365d6c2335a6026f55de443dd95c902f true
```

Comment:
https://access.redhat.com/solutions/6964609

we might be stuck :(

Reply:
The solution with an OLS+RAG+test, the way this branch was in the morning, should I think work.

Reply:
yeah sorry i just meant stuck in terms of direct podman usage.